functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py

```diff
@@ -0,0 +1,264 @@
+from testcases import *
+from nose_parameterized import parameterized
+import random, unittest
+#from framework.api.client import Client
+
+
+
+class Test(TestcasesBase):
+
+    def setUp(self):
+        super().setUp()
+        self.user="{}@itsyouonline".format(self.whoami) 
```

Comment:
Ahmed will create a general method in the framework file which will create an account and return the accound id. Use it